.github/workflows/e2e-tests.yaml

```diff
@@ -18,17 +18,19 @@ jobs:
       matrix:
         operating-system: [ ubuntu-latest ]
         php-version: [ '8.0' ]
+        dependency: [ locked ]
```

Comment:
do we need to include `lowest`, `highest` here? Otherwise, I think I don't get how the are execued then

Reply:
The idea of this PR was just to be a preparation step. Adding `lowest` and `highest` should be done in dedicated PR IMO as they will require additional changes (code-wise).

Reply:
Rephrased the PR description, sorry this was not clear enough